oauth2-useragent-core/src/main/java/com/microsoft/alm/oauth2/useragent/AuthorizationResponse.java

```diff
@@ -61,6 +61,10 @@ public static AuthorizationResponse fromString(final String s) throws Authorizat
             for (final String pair : pairs) {
                 final String[] nameAndValue = NAME_VALUE_SEPARATOR.split(pair, 2);
                 try {
+                    if (nameAndValue.length != 2) {

```

Comment:
Do you have sample output you received from an OAuth 2.0 service that we could use to write a test with?